gym/wrappers/common/__init__.py

```diff
@@ -0,0 +1 @@
+from gym.wrappers.common.frame_skipping import *
```

Comment:
Please explicitly import any symbols you need.

Also, I don't think that wrappers are going to group like environments. Please put them under `gym/wrappers/...`.